partials/footer.hbs

```diff
@@ -3,27 +3,96 @@ This footer template is shared across all the pages.
 --}}
 
 <footer class="m-footer">
-  <div class="m-footer__content">
-    <p class="m-footer-copyright">
+  <div class="footer-content row">
+    <div class="four-columns footer-columns">
+      <div class="footer-brand-div text-center">
+        <h2>University of California, Berkeley IEEE Student Branch</h2>
+        <p>251 CoryHall, University of California, Berkeley</p>
+      </div>
+    </div>
+
+    <div class="four-columns footer-columns">
+      <div class="text-center">
+        <h3 class="footer-heading">Links</h3>
+        <p>
+          <a href="#" class="footer-link">FAQ</a><br />
+          <a href="#" class="footer-link">Check-In</a><br />
+          <a href="#" class="footer-link">About Us</a><br />
+          <a href="#" class="footer-link">Sponsors</a><br />
+          <a href="#" class="footer-link">Resources</a>
+        </p>
+      </div>
+    </div>
+
+    <div class="four-columns footer-columns text-center">
+      <h3 class="footer-heading">For Officers</h3>
+      <p>
+          <a href="#" class="footer-link">Blog</a><br />
+          <a href="#" class="footer-link">Check In</a><br />
+          <a href="#" class="footer-link">Execs and Officers</a><br />
+        </p>
+    </div>
+
+    <div class="four-columns footer-columns text-center">
+      <h3 class="footer-heading">Contact Us</h3>
+      <nav class="m-footer-social">
+        {{#if @site.facebook}}
+          <a  class="footer-link" href="{{facebook_url @site.facebook}}" target="_blank" rel="noopener" aria-label="Facebook">
+            <span class="icon-facebook"></span>
+          </a>
+        {{/if}}
+        {{#if @site.twitter}}
+          <a  class="footer-link" href="{{twitter_url @site.twitter}}" target="_blank" rel="noopener" aria-label="Twitter">
+            <span class="icon-twitter"></span>
+          </a>
+        {{/if}}
+      </nav>
+     
+      <a href="/apply" type="button" class="ieee-btn link"><span class="footer-btn-txt">Join Our Mailing List</span></a>
```

Comment:
I would probably give a little more space between the mailing list button and the social media buttons